lib/components/form/date-time-modal.js

```diff
@@ -2,74 +2,126 @@
+    height: 34px;
+    padding: 6px 12px;
+    text-align: center;
+    &.focused {
+      outline: 0;
+      border-color: #66afe9;
+      font-weight: 400;
+      box-shadow: inset 0 3px 5px rgba(0, 0, 0, .125);
+    }
+  }
+
+  button {
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    background-color: #fff;
+    border: 1px solid #ccc;
+    border-radius: 4px;
+    color: #333;
+    cursor: pointer;
+    font-family: inherit;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 1.42857143;
+    outline-offset:-2px;
+    padding: 6px 12px;
+    text-align: center;
+    touch-action: manipulation;
+    user-select: none;
+    white-space: nowrap;
+
+    &.active {
+      background-color: #e6e6e6;
+      border-color: #adadad;
+      box-shadow: inset 0 3px 5px rgba(0, 0, 0, .125);
+      font-weight: 400;
+    }
+    &:hover {
+      background-color: #e6e6e6;
+      border-color: #adadad;
+    }
+    &.active {
+      background-color: #e6e6e6;
+      border-color: #adadad;
+      box-shadow: inset 0 3px 5px rgba(0, 0, 0, .125);
+      font-weight: 400;
+      &:hover {
+        background-color: #d4d4d4;
+        border-color: #8c8c8c;
+      }
+    }
+  }
+`
 
 class DateTimeModal extends Component {
   static propTypes = {
-    routingType: PropTypes.string,
     setQueryParam: PropTypes.func
   }
 
   render () {
-    const { config, routingType, setQueryParam } = this.props
+    const { date, dateFormatLegacy, departArrive, setQueryParam, time, timeFormatLegacy } = this.props
 
     return (
       <div className='date-time-modal'>
-        {/* The routing-type selection button row. Only show if more than one configured */}
-        {config.routingTypes.length > 1 && (
-          <div className='button-row'>
-            <ButtonGroup justified>
-              {config.routingTypes.map(rtConfig => {
-                return (
-                  <ButtonGroup key={rtConfig.key}>
-                    <Button
-                      className={rtConfig.key === routingType ? 'selected' : ''}
-                      onClick={() => {
-                        setQueryParam({ routingType: rtConfig.key })
-                      }}
-                    >
-                      {rtConfig.text || rtDefaults.find(d => d.key === rtConfig.key).text}
-                    </Button>
-                  </ButtonGroup>
-                )
-              })}
-            </ButtonGroup>
-          </div>
-        )}
-
-        {/* The main panel for the selected routing type */}
         <div className='main-panel'>
-          {rtDefaults.find(d => d.key === routingType).component}
+          <StyledDateTimeSelector
+            className='date-time-selector'
+            date={date}
+            dateFormatLegacy={dateFormatLegacy}
+            departArrive={departArrive}
+            onQueryParamChange={setQueryParam}
+            time={time}
+            timeFormatLegacy={timeFormatLegacy}
```

Comment:
Could you explain why we're using props called `*Legacy`? This kind of splitting of code paths within the components doesn't seem necessary -- we should not be considering the otp-react-redux library "legacy" software. I don't think I had a chance to review opentripplanner/otp-ui#32, otherwise, I wouldn't have supported the splitting of the handlers like this.

Reply:
I agree on adding more documentation to `DateTimeSelector` in OTP-ui. There is a tiny bit [here](https://github.com/opentripplanner/otp-ui/blob/master/packages/trip-form/src/DateTimeSelector/index.js#L213).

Reply:
See my adjusted the comments that are relevant to OTP-RR.

Reply:
I added additional docs on the OTP-UI side, see https://github.com/opentripplanner/otp-ui/pull/98